etc/systemd/system/zfs-import-scan.service.in

```diff
@@ -13,6 +13,7 @@ Type=oneshot
 RemainAfterExit=yes
 ExecStartPre=/sbin/modprobe zfs
 ExecStart=@sbindir@/zpool import -aN -o cachefile=none
+ExecStartPost=/bin/bash -c "/usr/bin/systemctl set-environment BOOTFS=$(@sbindir@/zpool list -H -o bootfs)"
```

Comment:
This could find the wrong bootfs, but since the find_bootfs function basically does the same thing there doesn't seem to be a better option.  It has no clue what's 'right' for zfs:AUTO, but we could try and import JUST the BOOTFS pool if it's not zfs:AUTO.  Do you think we should error out if BOOTFS is more than one line?